src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -61,13 +62,18 @@ public PersonCard(Person person, int displayedIndex) {
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
 
-        IntStream.rangeClosed(1, 10).forEach(index -> {
+        IntStream.rangeClosed(1, Person.MAXIMUM_TUTORIALS).forEach(index -> {
             Label tutorialLabel = new Label(String.valueOf(index));
             tutorialLabel.getStyleClass().add("tutorial-label");
 
-            if (person.hasAttendedTutorial(String.valueOf(index))) {
+            if (person.getAttendanceStatus(String.valueOf(index)) == AttendanceStatus.ATTENDED) {
```

Comment:
Should we have a method in person to do this check instead?

Reply:
Lifted the method up to the Person class to retrieve CSS class.